### DISCUSSION TITLE
Suitability of Anura3D/MPM for longer runouts

### DISCUSSION THREAD

**Discussion by mikael-onsilva** [Anura3D/Anura3D_OpenSource#150] [General]
Hello everyone,

I'm starting to study Anura3D to model some tailings stack failures, with the final goal of having the affected areas. The runout can be for just a few meters up to a few kilometers, depending on the topography and if the material is liquefiable or not.

For cases of larger deformations like these ones expected, is a MPM approach still valid?

Thank you all for the inputs.

**Comment by murphy-john** [comment]
This is a philosophical question which is not possible for us to answer especially in the context of a discussion forum. It's up to each modeler to decide which numerical tool is best suited for their problem. There are certain capabilities which make MPM well suited to this problem but until someone has tried and demonstrated, there is no way for anyone to know. That's why we do research.
